lib/RestructureCFG/EnforceSingleExitPass.cpp

```diff
@@ -55,9 +55,10 @@ static bool isInfiniteLoop(const scc_iterator<Scope<BasicBlock *>> &SCCIt) {
 /// blocks, and infinite loops too.
 class EnforceSingleExitPassImpl {
   Function &F;
+  ScopeGraphBuilder SGBuilder;
```

Comment:
Same here. It should be possible to make this `const`.

Reply:
I made this field `const` here https://github.com/revng/revng/blob/3a9379ef740d1d2b9f7bcad02cc962719251e02a/lib/RestructureCFG/EnforceSingleExitPass.cpp#L58